release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -25,7 +25,9 @@ You must use {op-system} machines for the control plane, and you can use either
 //Removed the note per https://issues.redhat.com/browse/GRPA-3517
 
 //TODO: Add this for 4.14
-Starting with {product-title} 4.12 an additional six months is added to the Extended Update Support (EUS) phase on even numbered releases from 18 months to two years. For more information, see the link:https://access.redhat.com/support/policy/updates/openshift[Red Hat {product-title} Life Cycle Policy].
+Starting with {product-title} 4.12, an additional six months is added to the Extended Update Support (EUS) phase on even numbered releases from 18 months to two years. For more information, see the link:https://access.redhat.com/support/policy/updates/openshift[Red Hat {product-title} Life Cycle Policy].
+
+Starting with {product-title} 4.14, Extended Update Support (EUS) is extended to 64-bit ARM, {ibmpowerProductName}(ppc64le), and {ibmzProductName}(s390x) platforms.  For more information, see the link:https://access.redhat.com/support/policy/updates/openshift-eus[OpenShift EUS Overview].
```

Comment:
Does EUS need to be spelled out again since it is spelled out in the previous paragraph?

Reply:
@JoeAldinger WDYT? 
Looking at the 4.12 entry (which was the last EUS release) it was spelled out every time. https://docs.openshift.com/container-platform/4.12/release_notes/ocp-4-12-release-notes.html#ocp-4-12-about-this-release

Reply:
@SNiemann15 I think we should keep things consistent, but also understand the position of not needing to cc @kcarmichael08. I'm in favor of being cautious here and spelling it out since this is a support statement and acronyms, I'm sure, are even more complicated in translation. What do you all think?